<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>804</width>
    <height>419</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>804</width>
    <height>419</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>NoLimit Math - SYMPY VERSION - by SM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icona.png</normaloff>../icona.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff0000;font-family:Rockwell;</string>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLim">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff0000;font-family:Rockwell;</string>
    </property>
    <property name="text">
     <string>lim</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:white;
color:darkred;
border:2px solid darkred;
border-radius:5px;
font-family:Rockwell;
}
QPushButton:hover
{
background-color:darkred;
color:white;
}
QPushButton:pressed
{
background-color:red;
border:0px;
}</string>
    </property>
    <property name="text">
     <string>Calculate and
generate the graph</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_plot"/>
   </widget>
   <widget class="QPushButton" name="show_details">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:white;
color:darkred;
border:2px solid darkred;
border-radius:5px;
font-family:Rockwell;
}
QPushButton:hover
{
background-color:darkred;
color:white;
}
QPushButton:pressed
{
background-color:red;
border:0px;
}</string>
    </property>
    <property name="text">
     <string>View the detailed graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>341</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:transparent;color:darkred;border:0px solid transparent;border-bottom:1px solid darkred;opacity:.7;font-family:Rockwell;} QLineEdit:focus{border-bottom:2px solid red;opacity:1;background-color:transparent;}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="limit_expression">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>281</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:transparent;color:darkred;border:0px solid transparent;border-bottom:1px solid darkred;opacity:.7;font-family:Rockwell;} QLineEdit:focus{border-bottom:2px solid red;opacity:1;background-color:transparent;}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="limit_value">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>135</y>
      <width>45</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{background-color:transparent;color:darkred;border:0px solid transparent;border-bottom:1px solid darkred;opacity:.7;font-family:Rockwell;} QLineEdit:focus{border-bottom:2px solid red;opacity:1;background-color:transparent;}</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff0000;font-family:Rockwell;</string>
    </property>
    <property name="text">
     <string>xâ†’</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>3</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:darkred;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>351</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:darkred;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="liveCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:darkred;
font-family:Rockwell;</string>
    </property>
    <property name="text">
     <string>Live calculation</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
background-image:url('../icona50x50.png');
background-repeat:no-repeat;
background-position:left center;
border:0px solid transparent;</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>351</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:darkred;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelLim_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff0000;font-family:Rockwell;font-size:26px;</string>
    </property>
    <property name="text">
     <string>NoLimit Math</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVersione">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff0000;font-family:Rockwell;font-size:26px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:16px; color:#ffffff; background-color:#8b0000;&quot;&gt;&amp;nbsp;v&amp;nbsp;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="resultFloat">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#ff0000;font-family:Rockwell;</string>
    </property>
    <property name="text">
     <string>Float</string>
    </property>
   </widget>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>&amp;about</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
